docs/monthly/2024-01-monthly.md

```diff
@@ -12,150 +12,97 @@ developments to gather their feedback on proposed plans.
-[roadmap](https://github.com/orgs/input-output-hk/projects/21/views/7):
+The Hydra team restructured several items on the project [roadmap](https://github.com/orgs/input-output-hk/projects/21/views/7) this month.
 
 ![The roadmap with features and ideas](./img/2024-01-roadmap.jpg) <small><center>The latest roadmap with features and ideas</center></small>
 
-#### Notable updates
+### Notable updates
 
-* Released version `0.15.0` which delivers offline mode and Conway support.
+* Released version `0.15.0`, delivering offline mode and Conway support.
 
-* [Offline mode #1254](https://github.com/input-output-hk/hydra/issues/1254) is
-  a new feature contributed by @SundaeLabs and is related to [this Catalyst
-  project](https://milestones.projectcatalyst.io/projects/1000179)
+* @SundaeLabs contributed a new feature, [offline mode #1254](https://github.com/input-output-hk/hydra/issues/1254), which is related to [this Catalyst project](https://milestones.projectcatalyst.io/projects/1000179).
 
-* Delivered [Conway support
-  #1177](https://github.com/input-output-hk/hydra/issues/1177) which will
-  prepare `hydra-node` for the upcoming hard-fork into the Conway era.
+* Delivered [Conway support #1177](https://github.com/input-output-hk/hydra/issues/1177) preparing `hydra-node` for the upcoming Conway hard fork.
 
-* New feature to indicate preparation of [running latest version cardano-nodes in P2P
-#1256](https://github.com/input-output-hk/hydra/issues/1256).
-  This is necessary as the non-P2P relay nodes of IOG are to be shut down in January 2024.
+* Implemented a new feature, indicating preparation for [running the latest version of Cardano nodes in P2P #1256](https://github.com/input-output-hk/hydra/issues/1256). This is necessary as IOG's non-P2P relay nodes are to be shut down in January 2024.
 
-* Similarly, new feature to [run smoke tests in Sanchonet
-  #1257](https://github.com/input-output-hk/hydra/issues/1257) which will serve
-  as another proof point of being able to open/close heads in the new Conway
-  era.
+* Worked on implementing the new feature, [running smoke tests in SanchoNet #1257](https://github.com/input-output-hk/hydra/issues/1257), which will provide additional evidence of the capability to open and close heads in the Conway era.
 
-* Groomed and started work on [building and deploying a Hydra heads explorer
-  #696](https://github.com/input-output-hk/hydra/issues/696), details below.
+* Groomed and started work on [building and deploying a Hydra heads explorer #696](https://github.com/input-output-hk/hydra/issues/696), see details below.
 
-* Renamed "Drop Babbage support #1178" to [Switch L2 ledger to Conway
-  #1178](https://github.com/input-output-hk/hydra/issues/1178) to better capture
-  it's content.
+* Renamed 'Drop Babbage support #1178' to [Switch layer 2 ledger to Conway #1178](https://github.com/input-output-hk/hydra/issues/1178) to better capture its content.
```

Comment:
@olgahryniuk @oduameh I see many line merges. Do you prefer markdown with fully connected paragraphs? It renders the same, but let's align on one format.

Reply:
I would so much appreciate connected lines and sentences. This would make working in suggestion mode much simpler.

Reply:
Fine by me, I'll connect all remaining ones.